### PR TITLE
Stampy redesign 376

### PR DESCRIPTION
**Pull request by varchanaiyer**

### CODE REVIEW

**Comment by mruwnik** [comment]
don't merge to master - the redesign branch is stampy-redesign

**Comment by mruwnik** [comment]
there are some conflicts that need to be fixed here

File: app/components/Chatbot/ChatEntry.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {ComponentType} from 'react'
-import {Link} from '@remix-run/react'
+import { ComponentType } from 'react'
```
Comment (melissasamworth) [inline app/components/Chatbot/ChatEntry.tsx:1]: What was the purpose of adding all these spaces?

**Review by melissasamworth — COMMENTED** [review]
Gah I am sorry but we cannot merge this! There are changes to the root.css file, which I am very protective of (I'm sorry, I should have expressed that I am really pedantic that no one touches the tags and one-off classes, particularly the typography classes).

Also there are too many conflicts. As a result I can't see what is changed. All I see is that we are missing the feedback widget under the human-written responses.

I would say I'm going to delete this request, and you can make a new one. So save the code you want to save! Hope that okay, I think saving it will just not be worth it at this point.

File: app/components/Chatbot/ChatEntry.tsx (hunk 1 above)
Comment (mruwnik) [reply app/components/Chatbot/ChatEntry.tsx:1]: probably auto import + the editor doing it. @varchanaiyer, make sure to run `npm run lint:fix` before pushing, as that will both make the code style consistent and notify you of any errors

**Pull request closed without merging**